_content/articles/living-with-the-Mountain_bhutia.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Living with the Mountain: Mountain Propitiation Rituals in the Making of Human-Environmental Ethics in Sikkim"
+authors:
+  - Kalzang Dorjee Bhutia
+external_url: "https://blogs.dickinson.edu/buddhistethics/files/2021/11/Bhutia_21_FD2-1.pdf"
+source_url: "https://blogs.dickinson.edu/buddhistethics/2021/11/27/mountain-propitiation-rituals-in-human-environmental-ethics-in-sikkim/"
+drive_links:
+  - "https://drive.google.com/file/d/16kmDCmkEVBSo-yMk07BCJBET8S9jr2ub/view?usp=share_link"
+formats: [pdf]
+course: nature 
+tags:
+  - modern-indian
+  - mountains
+  - ritual
+  - protective deities
+year: 2021
+journal: jbe
+volume: 28
+pages: "261--294"
+---
+This article shows the complications that arise when religous traditions come in contact with the challenges of the modern world. The government of Sikkim is underpressure to allow climbers to access the world's third highest mountain, Mount Kanchenjung, held to be very sacred to the local community as the dwelling place of a protective deity.
```

Comment:
Under pressure should be two words here. And I'd add something about the (anthropomorphic? agentive?) relationship with the environment mentioned here.